app/models/katello/host_collection.rb

```diff
@@ -120,7 +120,7 @@ def consumer_ids
     end
 
     def errata(type = nil)
-      query = Erratum.joins(:system_errata).where("#{SystemErratum.table_name}.system_id" => self.consumer_ids)
+      query = Erratum.joins(:content_facet_errata).joins(:content_facets).where("#{Katello::Host::ContentFacet.table_name}.uuid" => self.consumer_ids)
```

Comment:
why not just do where("#{Katello::Host::ContentFacet.table_name}.host_id" => self.host_ids)